test/unit/webdriver/device/islocked_test.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from test.unit.helper.test_helper import appium_command, android_w3c_driver, ios_w3c_driver
+
+import httpretty
+
+
+class TestWebDriverDeviceIsLocked(object):
+
+    @httpretty.activate
+    def test_islocked_false(self):
+        driver = android_w3c_driver()
+        httpretty.register_uri(
+            httpretty.POST,
+            appium_command('/session/1234567890/appium/device/is_locked'),
+            body='{"value": false}'
+        )
+        assert driver.is_locked() == False
```

Comment:
What about `assert xxxx is False` syntax

Reply:
What about assert xxxx is False syntax ?

Reply:
https://stackoverflow.com/questions/27276610/boolean-identity-true-vs-is-true
`is`, `is not`

Reply:
Thanks done